dataprofiler/profilers/text_column_profile.py

```diff
@@ -45,10 +44,14 @@ def __add__(self, other):
             raise TypeError("Unsupported operand type(s) for +: "
                             "'TextColumn' and '{}'".format(other.__class__.__name__))
         merged_profile = TextColumn(None)
-        merged_profile.vocab = self.vocab.copy()
-        merged_profile._update_vocab([other.vocab])
         NumericStatsMixin._add_helper(merged_profile, self, other)
         BaseColumnPrimitiveTypeProfiler._add_helper(merged_profile, self, other)
+        self._merge_calculations(merged_profile.__calculations,
+                                 self.__calculations,
+                                 other.__calculations)
+        if "vocab" in merged_profile.__calculations:
+            merged_profile.vocab = self.vocab.copy()
+            merged_profile._update_vocab([other.vocab])
```

Comment:
is this right that it should be a list of lists? or can it just be: `merged_profile._update_vocab(other.vocab)`

Reply:
This was the precedence set, so I'm not sure, I can try without and see what happens

Reply:
I took it out and it works. Idk why we had that